### DISCUSSION TITLE
openjdk 17 einsetzbar?

### DISCUSSION THREAD

**Discussion by klaus-schauer** [camueller/SmartApplianceEnabler#623] [Installation und Betrieb]
Ist SAE unter openjdk 17 statt openjdk 11 lauffähig?

**Comment by camueller** [comment]
Vermutlich, habe es nicht getestet.

**Comment by camueller** [comment]
Update: ja JDK 17 funktioniert. Die automatische Installation habe ich inzwischen auf JDK 17 umgestellt.
